### ISSUE TITLE
[BUG]: Can't download anything using drm_adaptive_dash - web/android

### ISSUE THREAD

**Issue by hazy-kun** [anidl/multi-downloader-nx#720]
### Program version

5.0.3

### Operating System

Windows

### Type

GUI

### Service

Crunchyroll

### Command used

{"id":"GRWEW95KR","q":0,"e":"7","dubLang":["jpn"],"dlsubs":["none"],"fileName":"[${service}] ${showTitle} S${season}E${episode} ${title}","dlVideoOnce":false,"all":false,"but":false,"noaudio":false,"novids":false,"simul":false,"ids":["G6095ZK26"],"title":"Mount Fuji and Curry Noodles","parent":{"title":"Laid-Back Camp","season":"1"},"image":"https://www.crunchyroll.com/imgsrv/display/thumbnail/1200x675/catalog/crunchyroll/a3b92b8b50f384e912abcc8125349b92.jpe","episode":"1"}

### Show ID

GRWEW95KR

### Episode

all

### Console Output

```Shell
[ERROR] 403: Forbidden
[ERROR] <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<HTML><HEAD><META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
<TITLE>ERROR: The request could not be satisfied</TITLE>
</HEAD><BODY>
<H1>403 ERROR</H1>
<H2>The request could not be satisfied.</H2>
<HR noshade size="1px">
Request blocked.
We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
<BR clear="all">
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation.
<BR clear="all">
<HR noshade size="1px">
<PRE>
Generated by cloudfront (CloudFront)
Request ID: hKITqnj7VPWaAXuSTYF7ncEYFbec507kHWqbTKrhTlkNeAALx7oOkA==
</PRE>
<ADDRESS>
</ADDRESS>
</BODY></HTML>
[ERROR] CAN'T FETCH VIDEO PLAYLISTS!
Subtitles downloading skipped!
[ERROR] Download error: Unable to download episode 7 from GRWEW95KR
```

**Comment by Jaynator495** [comment]
Yea, unfortunately it seems like crunchy has disabled that stream, I'm gonna leave it in the code for now incase it comes back, but next version will learn more into the Crunchy Play Streams.

Ironically though, they added DRM to the switch stream, so stream number 1 uses DRM now

**Comment by hazy-kun** [comment]
Yeah i know, Switch endpoint is now OMA enccrypted

**Comment by jakesteele** [comment]
I am also running into this issue.

**Comment by tormento** [comment]
It's still working here (with DRM obv).

**Comment by hazy-kun** [comment]
Seems that the only working setting is:
```
crapi: "web"
kstream: 1
```

**Comment by Jaynator495** [comment]
> Seems that the only working setting is:
> 
> ```
> crapi: "web"
> kstream: 1
> ```

Yea, all the other streams crunchy seems to have killed, next release will have them disabled for the time being

**Comment by AnimeDL** [comment]
Released version 5.1.0 which removes all non-working streams, and adds a way to switch between endpoints for the latest stream API, `--cstream`

**Comment by AnimeDL** [comment]
Going to close this now
